src/sidebars/favourites/minichart/minichart.less

```diff
@@ -38,8 +39,8 @@
 }
 
 .minichart-error {
-    padding: 0px 3px;
-    text-align: center;
-    .font(12px, @font-thin);
-    color: white;
+  padding: 0px 3px;
+  text-align: center;
+  .font(12px, @font-thin);
```

Comment:
I don't understand the comment. That is a mixin already. (Again, not trying to pass the buck here, but all I did was update the whitespace!)